<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Huffman</class>
 <widget class="QMainWindow" name="Huffman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huffman</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>240</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Code">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bottom_browser">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>210</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bottom_file">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select path...</string>
    </property>
   </widget>
   <widget class="QLabel" name="presentation_label">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>280</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Author: ≈Åukasz Plech Ti1 PK 2018/2019</string>
    </property>
   </widget>
   <widget class="QToolButton" name="top_browser">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>180</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="top_file">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select path...</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Decode">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>10</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="information_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Project: Huffman coding</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
